src/views/Stats.vue

```diff
@@ -89,6 +89,22 @@
     </v-col>
   </v-row>
 
+  <br>
+
+  <h2 class="text-h6 mb-1">
+    <v-icon size="x-small" icon="mdi-test-tube" />
+    {{ $t('Common.Experiments') }}
+  </h2>
+
+  <v-row>
+    <v-col cols="6" sm="4" md="3" lg="2">
+      <StatCard :value="1" :subtitle="$t('Common.Challenges')" />
```

Comment:
value hardcoded for now, until we start storing the Challenge metadata in a backend model